kubernetes/linux/setup.sh

```diff
@@ -29,6 +29,14 @@ rm -rf /usr/lib/ruby/gems/3.1.0/gems/openssl-3.0.1
 rm /usr/lib/ruby/gems/3.1.0/specifications/default/find-0.1.1.gemspec
 rm -rf /usr/lib/ruby/gems/3.1.0/gems/find-0.1.1
 
+# update the time and uri package to tackle the vulnerabilities in these gems
+gem update time --default
```

Comment:
is tehre any way to directly uninstall time and uri such as deleting the folder or use gem uninstall

Reply:
No we are not deleting the gems, we are updating the gem and deleting the old version

Reply:
can we uninstall first then upgrade to save the mv?

Reply:
No we cant uninstall default gem...So, what we are doing is upgrade, get new version, move the older files so the new file is now default...Then gem uninstall the older version